client-src/src/actions/fetch-utility-type-instances.js

```diff
@@ -0,0 +1,20 @@
+import axios from 'axios';
+import settings from '../settings';
+
+export default function fetchUtilityTypeInstances(e) {
+    const END_POINT = `${settings.hostname}/api/`;
+    const RESOURCE = `Utilities`;
+
+    const FILTER = {
+        where: {
+            utility_type_id: e.target.value
```

Comment:
same thing, let's just get the value here, this function isn't supposed to know _how_ to get the value